hedera-node/hedera-app/src/test/java/com/hedera/node/app/state/impl/StateRegistryTest.java

```diff
@@ -0,0 +1,24 @@
+package com.hedera.node.app.state.impl;
+
+import com.hedera.node.app.state.merkle.HederaStateImpl;
+import com.hedera.node.app.state.merkle.ServiceStateNode;
+import org.junit.jupiter.api.Test;
+
+class StateRegistryTest {
+    @Test
+    void foo() {
+        // Create the merkle state tree
+        final var state = new HederaStateImpl();
+
+        // Create a fake "consensus service" and add it to the merkle tree
+        final var consensusService = new ServiceStateNode("Consensus Service");
+        state.addServiceStateNode(consensusService);
+
+        // Now, pretend we're going to create a StateRegistry to pass to the ConsensusService
+        // in its constructor, and let it create the "topics" merkle map.
+        final var registry = new StateRegistryImpl(consensusService);
+//        registry.registerOrMigrate("topics", (definer, opt) -> {
```

Comment:
uncomment it or delete it. Or add a comment why the code is commented

Reply:
Will do. This test is not yet complete, I will finish it.